ocaml/tests/test_vm_migrate.ml

```diff
@@ -12,57 +12,62 @@
  * GNU Lesser General Public License for more details.
  *)
 
-open OUnit
-open Test_common
-
 let mac1 = "00:00:00:00:00:01"
 let mac2 = "00:00:00:00:00:02"
 
+let check_network_map =
+  Alcotest.(check (list
+                  (pair
+                    (Alcotest_comparators.ref ())
+                    (Alcotest_comparators.ref ()))))
+
 let test_infer_vif_map_empty () =
-  let __context = make_test_database () in
-  assert_equal
+  let __context = Test_common.make_test_database () in
+  check_network_map "Asserted by test_infer_vif_map_empty"
     (Xapi_vm_migrate.infer_vif_map ~__context [] [])
     []
 
 let test_infer_vif_map () =
-  let __context = make_test_database () in
-  let vm_vif1 = make_vif ~__context ~mAC:mac1 () in
-  let vm_vif2 = make_vif ~__context ~mAC:mac2 () in
-  let snap_vif1 = make_vif ~__context ~mAC:mac1 () in
-  let snap_vif2 = make_vif ~__context ~mAC:mac2 () in
+  let __context = Test_common.make_test_database () in
+  let vm_vif1 = Test_common.make_vif ~__context ~mAC:mac1 () in
+  let vm_vif2 = Test_common.make_vif ~__context ~mAC:mac2 () in
+  let snap_vif1 = Test_common.make_vif ~__context ~mAC:mac1 () in
+  let snap_vif2 = Test_common.make_vif ~__context ~mAC:mac2 () in
   (* In reality this network won't be in the local database, but for our
      	 * purposes it is a meaningless UUID so it's OK for it to be in the local
      	 * database. *)
-  let network1 = make_network ~__context () in
+  let network1 = Test_common.make_network ~__context () in
   (* Check that a map with a single VIF -> network pair is unchanged. *)
```

Comment:
I think we can remove these comments now as they are part of the code :) (To avoid duplicating them)